client/data/settings/selectors.js

```diff
@@ -121,10 +121,6 @@ export const getDepositScheduleInterval = ( state ) => {
 	return getSettings( state ).deposit_schedule_interval || '';
 };
 
```

Comment:
Note that it looks like a bunch of code related to the now-redundant `reporting_export_language` can also be removed from https://github.com/Automattic/woocommerce-payments/blob/2af6f1aed5923631f49a72f1f98cad7a3c2256ad/includes/admin/class-wc-rest-payments-settings-controller.php.

Reply:
That is a nice catch! More 🪚 cuts! 
I have removed it here [edaaef9](https://github.com/Automattic/woocommerce-payments/pull/10003/commits/edaaef9b3ae18c7699629fac724c4b206c20aa2e)